app/serializers/user_post_topic_bookmark_base_serializer.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require_relative 'post_item_excerpt'
+
+class UserPostTopicBookmarkBaseSerializer < UserBookmarkBaseSerializer
+  include TopicTagsMixin
+  include PostItemExcerpt
+
+  attributes :topic_id,
+             :linked_post_number,
+             :title,
+             :fancy_title,
```

Comment:
These two attributes can be removed as they have been declared in the parent serializer.

Reply:
Ah yep will remove.